WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/ScanViewModel.kt

```diff
@@ -120,33 +122,33 @@ class ScanViewModel @Inject constructor(
     private fun fetchFixThreatsStatus(fixableThreatIds: List<Long>) {
         launch {
             @StringRes var messageRes: Int? = null
-            var isFixing: Boolean
+            var fixingThreatIds: List<Long>
+            val scanStateModel = requireNotNull(scanStore.getScanStateForSite(site))
             fetchFixThreatsStatusUseCase.fetchFixThreatsStatus(
                 remoteSiteId = site.siteId,
                 fixableThreatIds = fixableThreatIds
             ).collect { status ->
+                fixingThreatIds = emptyList()
```

Comment:
Minor (🔍): Consider changing that to `var fixingThreatIds = emptyList<Long>()` and thus avoid having it being declared further up, thus saving a line. Wdyt?

Reply:
Progress state can change from `InProgress` to `Complete` or `Failure` states, `fixingThreatIds` should be set to empty in all these cases. Instead of putting it in every when case, I added it to the top of the block that executes on flow result.
With this context, let me know if you have a better idea about its placement 😅.

Reply:
Thank you Ashita, sorry that I wasn't clear enough! 🤗

Reply:
Nope, I misunderstood. Thank you for the review 🌟.